mne/preprocessing/tests/test_xdawn.py

```diff
@@ -262,4 +266,98 @@ def test_XdawnTransformer():
     pytest.raises(ValueError, xdt.inverse_transform, 42)
 
 
+def _simulate_erplike_mixed_data(n_epochs=100, n_channels=10):
+    rs = np.random.RandomState(42)
+    epoch_tmin, epoch_tmax = 0, 1
+    sfreq = 100
+    informative_ch_idx = 0
+
+    labels = rs.randint(0, 2, n_epochs)
+    n_times = int((epoch_tmax - epoch_tmin) * sfreq)
+    epoch_times = np.linspace(epoch_tmin, epoch_tmax, n_times)
+
+    target_template = 1e-6 * (epoch_times - epoch_tmax) * np.sin(
+        2 * np.pi * epoch_times)
+    nontarget_template = 0.7e-6 * (epoch_times - epoch_tmax) * np.sin(
+        2 * np.pi * (epoch_times - 0.1))
+
+    epoch_data = rs.randn(n_epochs, n_channels, n_times) * 5e-7
+    epoch_data[labels == 0, informative_ch_idx, :] += nontarget_template
+    epoch_data[labels == 1, informative_ch_idx, :] += target_template
+
+    mixing_mat = np.linalg.svd(rs.randn(n_channels, n_channels))[0]
```

Comment:
after doing `from scipy import linalg`

```suggestion
    mixing_mat = linalg.svd(rs.randn(n_channels, n_channels))[0]
```